app/models/inquiry/column.rb

```diff
@@ -83,4 +85,22 @@ def validate_check_box(answer, data)
       end
     end
   end
+
+  def validate_upload_file(answer, data)
+    # MegaBytes >> Bytes
+    if self.max_upload_file_size.to_i > 0 
+      fileSize  = data.values[2].to_i
+      limitSize = (self.max_upload_file_size * 1024 * 1024).to_i
+
+      if data.present? && data.value.present?
+        if fileSize > limitSize
+          answer.errors.add :base, "#{name}#{I18n.t(
+              "errors.messages.too_large_file",

```

Comment:
__[rubocop]__
* Indent the first parameter one step more than the start of the previous line.

<!-- Comment by [SideCI](https://sideci.com) -->